image-index.md

```diff
@@ -55,19 +55,29 @@ For the media type(s) that this document is compatible with, see the [matrix][ma
 
     - **`os.version`** *string*
 
-        This OPTIONAL property specifies the operating system version, for example `10.0.10586`.
+        This OPTIONAL property specifies the operating system version.
```

Comment:
> If not, I think it is even ok to mark this property as "reserved, SHOULD NOT use".

That sounds good to me (or just removing the property from the spec entirely).  Looking up docs for `GetVersionEx()` turned up [this][1]:

> `GetVersionEx` may be altered or unavailable for releases after Windows 8.1. Instead, use the Version Helper functions

The version helper functions [all look like comparisons][2], and poking around a bit [turns up][3]:

> Rather than using GetVersionEx to determine the operating system platform or version number, test for the presence of the feature itself.

So it seems like the recommended Windows practice would be to drop `os.version` and instead use `os.features`.

[1]: https://msdn.microsoft.com/en-us/library/windows/desktop/ms724451%28v=vs.85%29.aspx
[2]: https://msdn.microsoft.com/en-us/library/windows/desktop/dn424972%28v=vs.85%29.aspx
[3]: https://msdn.microsoft.com/en-us/library/windows/desktop/ms725492%28v=vs.85%29.aspx